qa/L0_state_management/test.sh

```diff
@@ -32,7 +32,6 @@ CLIENT_PROTOCOL="http"
 PORTS=(`find_available_ports 3`)
 GPUS=(`get_all_gpus_uuids`)
 OUTPUT_MODEL_REPOSITORY=${OUTPUT_MODEL_REPOSITORY:=`get_output_directory`}
-rm -rf $OUTPUT_MODEL_REPOSITORY
```

Comment:
Why remove this line?

Reply:
It is unnecessary because we are using `--override-output-model-repository`